docs/openapi-docs/src/test/resources/expected_multi_body_for_status_codes.yml

```diff
@@ -0,0 +1,51 @@
+openapi: 3.0.1
+info:
+  title: Fruits
+  version: '1.0'
+paths:
+  /:
+    get:
+      operationId: getRoot
+      responses:
+        '200':
+          description: ''
+        '404':
+          description: ''
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: '#/components/schemas/NotFound'
+                  - $ref: '#/components/schemas/Unauthorized'
+        default:
+          description: unknown
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Unknown'
+components:
+  schemas:
+    NotFound:
+      required:
+        - what
+      type: object
+      properties:
+        what:
+          type: string
+    Unauthorized:
```

Comment:
We can give description to this as well right ?

Reply:
It looks like according to the spec we should be able to do so.

Reply:
perfect

Reply:
Including it to the test